docs/Users_Guide/appendixH.rst

```diff
@@ -0,0 +1,87 @@
+
+Required Unstructured Grid Metadata
+===================================
+
+To correctly parse the UGRID topology, MET needs the following information that must be contained within the UGRID coordinates file or the UGRID data file. It is often the case that the variable names for these elements differ between UGRIDs. If the user is not using one of the supported UGRIDs, they will need to define the **ugrid_metadata_map** in a custom configuration file that they provide on the command line using the **-ugrid_config** command line argument, which tells MET the variable names the correspond to the following elements:
+
+.. _table_ugrid_metadata:
+
+.. list-table:: Required UGRID Metadata
+  :widths: auto
+  :header-rows: 1
+
+  * - Metadata Map Key
+    - Description
+    - Required
+  * - dim_face
+    - Dimension name for UGRID cells
+    - Required
+  * - lat_face
+    - Coordinate variable name for the latitude of each UGRID cell
+    - Required
+  * - lon_face
+    - Coordinate variable name for the longitude of each UGRID cell
+    - Required
+  * - dim_time
+    - Dimension name for the time coordinate
+    - Required
+  * - time
+    - Coordinate variable for time
+    - Required
+  * - dim_vert
+    - Dimension name for the vertical coordinate, if present
+    - Optional
+  * - vert_face
+    - Coordinate variable name for the vertical coordinate
+    - Optional
+
+Unstructured Grid Configuration Entries
+=======================================
+
+In the PointStat and GridStat config files, a user can control aspects of the UGRID capability. Since the UGRID support in MET is optional, these items are not included in the default MET config files. If the user requires modification for any of these, they must add them to their MET config file.
+
+ugrid_dataset
+-------------
+
+If the UGRID dataset is supported by MET, then this item is set to the string identifying the UGRID. Currently supported ugrid_dataset options include “lfric”, and “mpas”. If this item is not set, the user must provide a separate UGRID configuration file on the command line using the **-ugrid_config** command line argument.
+
+ugrid_max_distance_km
+---------------------
+
+For PointStat, this is the distance from each UGRID cell center that PointStat will search outward to collect observations to use for verification. The default is 0 km, which by default will use the closest observation to the UGRID cell for verification. For GridStat, this is the distance from each UGRID forecast cell center to search for observation grid cells to include when interpolating to the UGRID forecast cell locations. Currently, only the nearest point observation or gridded observation cell is used.
```

Comment:
In the GridStat part, it seems to imply the UGRID forecast cell center will be the location that is verified. But it will be interpolated to a structured grid first. Therefore, I suggest changing "UGRID forecast cell center" and "UGRID forecast cell" to say "forecast grid point", or something like that. Optionally you could add a note to see the limitations about GridStat interpolating the UGRID to a structured grid?